test/tests/AelinDeal.t.sol

```diff
@@ -79,7 +79,7 @@ contract AelinDealTest is Test {
 
         assertEq(AelinDeal(dealAddress).name(), "aeDeal-POOL");
         assertEq(AelinDeal(dealAddress).symbol(), "aeD-POOL");
-        assertEq(AelinDeal(dealAddress).decimals(), 18);
+        // assertEq(AelinDeal(dealAddress).decimals(), 18);
```

Comment:
Had to comment this otherwise I'd get build errors

Reply:
AelinDeal is now a ERC721 token contract so there's no `decimals()` function anymore. You can remove it